CHANGELOG.md

```diff
@@ -11,18 +11,29 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 ## Unreleased
 
+## [v0.11.0-rc.0](https://github.com/thanos-io/thanos/releases/tag/v0.11.0-rc.0) - 2020.02.19
+
 ### Fixed
 
-- [#2033](https://github.com/thanos-io/thanos/pull/2033) minio-go: Fixed Issue #1494 support Web Identity providers for IAM credentials for AWS EKS
-- [#1985](https://github.com/thanos-io/thanos/pull/1985) store gateway: Fixed case where series entry is larger than 64KB in index.
-- [#2051](https://github.com/thanos-io/thanos/pull/2051) ruler: Fixed issue where ruler does not expose shipper metrics.
-- [#2101](https://github.com/thanos-io/thanos/pull/2101) ruler: Fixed bug where thanos_alert_sender_errors_total was not registered.
+- [#2033](https://github.com/thanos-io/thanos/pull/2033) Minio-go: Fixed Issue #1494 support Web Identity providers for IAM credentials for AWS EKS
+- [#1985](https://github.com/thanos-io/thanos/pull/1985) Store Gateway: Fixed case where series entry is larger than 64KB in index.
+- [#2051](https://github.com/thanos-io/thanos/pull/2051) Ruler: Fixed issue where ruler does not expose shipper metrics.
+- [#2101](https://github.com/thanos-io/thanos/pull/2101) Ruler: Fixed bug where thanos_alert_sender_errors_total was not registered.
+- [#1789](https://github.com/thanos-io/thanos/pull/1789) Store Gateway: Improve timeouts
+- [#2009](https://github.com/thanos-io/thanos/pull/2009) Store Gateway: Add consistency delay to fetch blocks
```

Comment:
Can we elaborate a bit more? It worth to mention that this introduce the 30m delay by default for all new fresh(from Prometheus) blocks to appear in store gateway.

It is also "added". And let's mention the reason: it's to handle eventual consistent storages.

Reply:
`Minimum age of all blocks before they are being read. Set it to a safe value (e.g 30m) if your object storage is eventually consistent. GCS and S3 are (roughly) strongly consistent.`